assets/prometheus-k8s/rules.yaml

```diff
@@ -356,6 +356,8 @@ spec:
         label_node_hyperthread_enabled, label_node_openshift_io_os_id,label_node_role_kubernetes_io_master,label_node_role_kubernetes_io_infra)
         ) by (label_kubernetes_io_arch, label_node_hyperthread_enabled, label_node_openshift_io_os_id,label_node_role_kubernetes_io_master,label_node_role_kubernetes_io_infra)
       record: node_role_os_version_machine:cpu_capacity_sockets:sum
+    - expr: clamp_max(sum(alertmanager_notifications_total),1)
+      record: alertmanager_routing_enabled
```

Comment:
This doesn’t follow naming best practices.

Reply:
Do you think something like `cluster:alertmanager_routing_enabled:max` would be better?

Reply:
yes